addons/calendar/data/mail_template_data.xml

```diff
@@ -132,121 +127,116 @@
-                </t>
-                <li>Attendees
-                <ul>
-                    <li t-foreach="object.event_id.attendee_ids" t-as="attendee">
-                        <div t-attf-style="display: inline-block; border-radius: 50%; width: 10px; height: 10px; background: {{ colors.get(attendee.state) or 'white' }};"> </div>
-                        <t t-if="attendee.common_name != object.common_name">
-                            <span style="margin-left:5px" t-out="attendee.common_name or ''">Mitchell Admin</span>
-                        </t>
-                        <t t-else="">
-                            <span style="margin-left:5px">You</span>
-                        </t>
-                    </li>
-                </ul></li>
-                <t t-if="object.event_id.videocall_location">
-                    <li>
-                        How to Join:
-                        <t t-if="object.get_base_url() in object.event_id.videocall_location"> Join with Odoo Discuss</t>
-                        <t t-else=""> Join at</t><br/>
-                        <a t-att-href="object.event_id.videocall_location" target="_blank" t-out="object.event_id.videocall_location or ''">www.mycompany.com/calendar/join_videocall/xyz</a>
-                    </li>
-                </t>
-                <t t-if="not is_html_empty(object.event_id.description)">
-                    <li>Description of the event:
-                    <t t-out="object.event_id.description">Internal meeting for discussion for new pricing for product and services.</t></li>
-                </t>
-            </ul>
-        </td>
-    </tr></table>
-    <br/>
-    Thank you,
-    <t t-if="object.event_id.user_id.signature">
-        <br />
-        <div>--<br/><t t-out="object.event_id.user_id.signature or ''">Mitchell Admin</t></div>
-    </t>
+        </div>
+        <div t-if="object.event_id.videocall_location" style="margin-top: 5px;">
+            <span style="display: inline-block; width: 100px; opacity: 0.76;">
+                <t t-if="object.event_id.videocall_source == 'discuss'">Join with</t>
+                <t t-else="">Join</t>
+            </span>
+            <a t-attf-href="object.event_id.videocall_location" target="_blank">
+                <t t-if="object.event_id.videocall_source == 'discuss'">Odoo Discuss</t>
+                <t t-else="">Video meeting</t>
+            </a>
+        </div>
+    </div>
+    <div style="margin: 32px 0 0;">
+        <h2 t-attf-style="margin-bottom: 12px; font-size: 14px; font-weight: bold; color: {{user.company_id.email_secondary_color or '#875A7B'}};">Attendees</h2>
+        <div t-foreach="object.event_id.attendee_ids" t-as="attendee" style="display: flex; align-items: center; margin-bottom: 5px;">
+            <img t-attf-src="/calendar/static/src/img/state_{{ attendee.state }}.png" style="display: inline-block; border:0; outline:none; width: 15px; height: 15px;" />
```

Comment:
maybe `attendee.state or 'tentative'` to display a question mark if the attendee has no state ? I see we had a placeholder value before